packages/main/src/plugin/image-registry.ts

```diff
@@ -872,6 +874,17 @@ export class ImageRegistry {
       throw Error('Unable to validate provided credentials.');
     }
   }
+
+  async searchImages(registryName: string, term: string, limit: number): Promise<ImageSearchResult[]> {
```

Comment:
can we omit the registryName and assume default value is dockerhub ?

Also maybe we should have a single variable as parameter like searchOptions

Reply:
searchOptions?: {
registry?: string
query (rather than term)
limit (also should be optional and we default to like 20 or 30)